.eslintrc

```diff
@@ -0,0 +1,43 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6":true,
+        "mocha": true
+    },
+    "extends": [ "standard"],
```

Comment:
`electron-forge` recommends working from the `airbnb` standard, but I've not got strong feels on this so I left `standard` in here for now.